src/templates/interest/html.mjml

```diff
@@ -0,0 +1,78 @@
+<mjml owa="desktop" version="4.6.3">
+  <mj-head>
+    <mj-title>The {{ poolYear }} Confidence Pool is Now Open
+    </mj-title>
+    <mj-preview>{{{formatPreview 'The {{ poolYear }} confidence pool is open and ready for registration.'}}}</mj-preview>
+    {{> styles }}
+    <mj-style>
+      .subtitle {
+        font-weight: bold;
+        font-size: 19px;
+        margin-bottom: 0px;
+        margin-top: 25px;
+      }
+
+      .subtitle+p {
+        margin-top: 8px;
+      }
+    </mj-style>
+  </mj-head>
+  <mj-body>
+    {{> header }}
+
+    <mj-section padding="0px" text-align="center">
+      <mj-column width="100%" background-color="#fff" border-radius="10px">
+        <mj-text padding="10px 20px 10px 20px" font-size="18px" align="left" line-height="1.2">
+          <p>Hello,</p>
```

Comment:
@balexander82 Do we want to put first name here?  Everyone we are inviting will have a first name since they are in the users table.